### PR TITLE
Added Session 14

### PR DESCRIPTION
**Pull request by javarb**
Hi, 

 could you please review my PR for the last Session. 

Thank you!

### CODE REVIEW

File: sessions/2018/october/Session_14.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+|`/api/customer` |GET| Give me all customers. Hardcoded|
+|`/api/book`| POST | Save to an in-memory map in a service. Return a json with the ID of the newly created book.|
+|`/api/:id` |GET| Get by ID, should return a previously posted book|
+
+Also, write Integration tests
+
+**TIP**: Following, an example for the POST /book map:
+```java
+@GetMapping("/{id}")
+public MyPojo getData(@PathVariable("id") String id) {
```
Comment (kinbiko) [inline sessions/2018/october/Session_14.md]: Could you write in your own words what `@PathVariable` does?

File: sessions/2018/september/Session_14.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+### Notes:
+
+#### Demo application
+Roger was teaching me how to to develop a basic API web application in Spring using [Spring initializr][1].
+
+Some Spring annotations ([there are many][2]) were used in this session. A brief explanation of them and his role in the workflow of the Spring application can be seen below:
+
+- ##### [@SpringBootApplication][3]
+
+Spring Initializr adds `@SpringBootApplication` annotation to the main class (same as `@Configuration` `@EnableAutoConfiguration` `@ComponentScan`). This annotation make possible to found controllers, components and dependencies. Also allows to make a runnable web application:
```
[the remark this answers is not in the document]
Comment (kinbiko) [reply sessions/2018/september/Session_14.md]: Looks pretty accurate to me.

File: sessions/2018/september/Session_14.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- ##### [@Service][6]
+
+Classes denoted with `@Service` annotation are intented to perform bussiness logic:
+```java
+@Service
+public class DemoService {
+  ...
+}
+```
+Tasks that controller delegates such as validation are common in this kind of classes, whereas database callings or handling HTTP endpoints aren't.
```
[the remark this answers is not in the document]
Comment (kinbiko) [reply sessions/2018/september/Session_14.md]: Yes, that's right.

File: sessions/2018/september/Session_14.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+##### Note about databases:
+In Spring, the data we get back from the database is the model.
+
+There is _classes for getting the data from the database_, these classes (making the database calls) are annotated with [`@Repository`][6] and might have complicated methods for getting the right data out. E.g: `getProjectsForUsers(List<Long> userIds, int pageCount, int pageIndex)`.
+
+Also exists _data classes_ that has fields, getters and setters. No complicated methods
+
+In other frameworks like Ruby on Rails, the _data classes_ and the _classes for getting the data from the database_ are the same classes, so they could be called indistinguishably "the model" in the MVC approach.
```
[the remark this answers is not in the document]
Comment (kinbiko) [reply sessions/2018/september/Session_14.md]: That's correct.

File: sessions/2018/october/Session_14.md (hunk 1 above)
Comment (javarb) [reply sessions/2018/october/Session_14.md]: Done, thanks!

**Pull request merged**